utils/ansible/ftest/templates/daos-make.sh.j2

```diff
@@ -196,21 +204,49 @@ if [[ ! -d "$DAOS_INSTALL_DIR" ]] ; then
 	fi
 fi
 
-cd "$DAOS_SOURCE_DIR"
-SCONS_OPTS="-j $JOBS_NB --implicit-deps-changed"
-case $CMD in
-	install) SCONS_OPTS="$SCONS_OPTS --build-deps=yes --config=force" ;;
-	update) SCONS_OPTS="$SCONS_OPTS --build-deps=no" ;;
-	*) fatal "Unexpected case" ;;
-esac
-
-info "Building and installing DAOS from source tree $PWD to $DAOS_INSTALL_DIR"
-if ! run env MPI_PKG=any "$SCONS_EXE" BUILD_TYPE=debug PREFIX="$DAOS_INSTALL_DIR" install $SCONS_OPTS ; then
-	fatal "DAOS could not be properly build and/or installed"
+if [[ ! -d "$DAOS_VENV_DIR" ]] ; then
+	info "Creating DAOS python virtualenv directory $DAOS_VENV_DIR"
+	if ! run mkdir -p "$DAOS_VENV_DIR" ; then
+		fatal "DAOS python virtualenv directory $DAOS_VENV_DIR could not be created"
+	fi
+fi
+
+if [[ ! -f "$DAOS_VENV_DIR/bin/activate" ]] ; then
+	info "Creating DAOS python virtualenv $DAOS_VENV_DIR"
+	run $PYTHON_EXE -m venv "$DAOS_VENV_DIR"
+fi
+info "Installing DAOS python virtualenv $DAOS_VENV_DIR"
+source "$DAOS_VENV_DIR/bin/activate"
+run $PIP_EXE install --upgrade pip
+run $PIP_EXE install -r "$DAOS_SOURCE_DIR/requirements.txt"
+
+SCONS_OPTS="--directory="$DAOS_SOURCE_DIR" --jobs=$JOBS_NB"
+
+if "$DAOS_BUILD_DEPS" ; then
+	info "Building DAOS dependencies from source tree $DAOS_SOURCE_DIR"
+	if ! run env MPI_PKG=any "$SCONS_EXE" BUILD_TYPE="$DAOS_BUILD_TYPE" PREFIX="$DAOS_INSTALL_DIR" $SCONS_OPTS --build-deps=only ; then
```

Comment:
These options aren't used here.
```suggestion
	if ! run "$SCONS_EXE" SCONS_OPTS --build-deps=only ; then
```

Reply:
Are you sure PREFIX and BUILD_TYPE are not used.
I was expecting that at least these two variables to be used by the "configure" script of the dependencies.

Reply:
You're probably right.  There's a DEPS_BUILD_TYPE but now that I think about it maybe the deps use BUILD_TYPE and the build uses BUILD_TYPE and also DEPS_BUILD_TYPE if that was different.

Reply:
> > You're probably right. There's a DEPS_BUILD_TYPE but now that I think about it maybe the deps use BUILD_TYPE and the build uses BUILD_TYPE and also DEPS_BUILD_TYPE if that was different.
> 
> * [x]  Remove useless env

Fixed with commit 97ec61dff